src/templates/admin/elements/metadata.html

```diff
@@ -2,8 +2,9 @@
 {% load roles %}
 
 {% user_has_role request 'editor' as user_is_editor %}
+{% user_has_role request 'production' as user_is_production %}
 
-{% if not view_type == "author" and user_is_editor %}
+{% if not view_type == "author" and user_is_editor or user_is_production %}
```

Comment:
Okay I;m a bit confused here:
Why do we still have the `if not view_type == "author"` check?

Reply:
Commit added to remove it, with the new check it is vestigial.